.gitignore

```diff
@@ -24,6 +24,32 @@ manifest.mf
 npm-debug.log
 core/src/main/web/app/templates.js
 core/config/hash
-core/src/main/web/vendor/
 core/src/main/web/index.html
 /bin/
+
+core/src/main/web/vendor/bower_components/jquery/test
+core/src/main/web/vendor/bower_components/jquery/speed
+core/src/main/web/vendor/bower_components/jquery/src
+
+core/src/main/web/vendor/bower_components/q/design
+core/src/main/web/vendor/bower_components/q/examples
+core/src/main/web/vendor/bower_components/q/spec
+
+core/src/main/web/vendor/bower_components/font-awesome/src
+core/src/main/web/vendor/bower_components/font-awesome/less
+core/src/main/web/vendor/bower_components/font-awesome/scss
+
+core/src/main/web/vendor/bower_components/acorn/test
+
+core/src/main/web/vendor/bower_components/underscore.string/test
+
+core/src/main/web/vendor/DataTables-1.10.7/extensions/ColVis
```

Comment:
I am ignoring `extensions/ColVis` and several other folders within DataTables which we are not using anything out of.

DataTables ships with tons and tons of files, I ignore the ones that we are not using to prevent unneeded bloat to the repo.